af/fastpath/fastpath/core.py

```diff
@@ -1080,7 +1080,9 @@ def score_psiphon(msm) -> dict:
         logbug(0, "no resolver_ip", msm)
         scores["accuracy"] = 0.0
 
-    scores["extra"] = dict(test_runtime=msm.get("test_runtime"))
+    truntime = msm.get("test_runtime")
+    btime = tk.get("bootstrap_time")
```

Comment:
Why don't you instead use the `bootstrap_time` variable that you already have available from above? I think it's more robust to use that since it's doing a `get` with a default value so it won't cause an exception if the key is missing.

Reply:
I though we assumed it's there and wanted to raise rather than set to 0. But we can instead set it to 0 and rely on accuracy being set to 0.